packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -21,8 +21,6 @@ export enum DataListWrapModifier {
 }
 
 export interface SelectableRowObject {
-  /** Determines if only one of the selectable rows should be selectable at a time */
-  type: 'multiple' | 'single';
```

Comment:
This feels like a breaking change. Maybe we can just console log a warning if they use 'multiple' as the value and make a note in the prop description that it's always single?

Reply:
I'm fine with doing that instead, but I thought we could make breaking changes to beta props?

Reply:
op - you're right. I'm fine with it then :)